src/User/ProfilePage.jsx

```diff
@@ -78,6 +81,14 @@ const ProfilePage = ({
         <p className="occupation">{`Occupation: ${occupation}`}</p>
         {isCurrentUser && displayManager()}
       </StyledCard>
+      {isCurrentUser && (
+        <StyledCard className="employeesCard">
+          <StyledCardHeading className="employeeListHeader">
+            Employees:
+          </StyledCardHeading>
+          {employees.length > 0 && <EmployeeList employees={employees} />}
```

Comment:
Should this check be at a higher level? If you have 0 employees under your management it would still say `Employees:` Could potentially use a ternary operator to display something like `No linked employees`.

Reply:
Good idea, I didn't want the card to disappear altogether so that would solve it.